From 87d1f2c7702527023c187811c093b1b8f286b879 Mon Sep 17 00:00:00 2001
From: Ricky CX Wu <ricky.cx.wu.wiwynn@gmail.com>
Date: Thu, 1 Aug 2024 14:15:05 +0800
Subject: [PATCH] configurations: yosemite4: set fail safe percent to 90% for
 Fan fail

Gerrit: 73176.

Set fail safe percent to 90% for fan sensors so that pid-control service
would set fan PWM to 90% when one of the fan failed.

Note: Need to apply Gerrit 67566 before adding this change.

Tested:
Tested with patch of Gerrit 67566 and checked that the fan PWM would be
set to 90% when one fan failed.

Change-Id: I38acd26f2610ed0ae9dd95fc2130b124383b0be1
Signed-off-by: Ricky CX Wu <ricky.cx.wu.wiwynn@gmail.com>
---
 configurations/yosemite4.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configurations/yosemite4.json b/configurations/yosemite4.json
index b2e82f4..7e839b8 100644
--- a/configurations/yosemite4.json
+++ b/configurations/yosemite4.json
@@ -438,6 +438,7 @@
             "Class": "fan",
             "FFGainCoefficient": 1.0,
             "FFOffCoefficient": 0.0,
+            "FailSafePercent": 90.0,
             "ICoefficient": 0.0,
             "ILimitMax": 0.0,
             "ILimitMin": 0.0,
-- 
2.25.1

